compiler/msgs.nim

```diff
@@ -537,7 +545,7 @@ proc liMessage*(conf: ConfigRef; info: TLineInfo, msg: TMsgKind, arg: string,
       else:
         styledMsgWriteln(styleBright, loc, resetStyle, color, title, resetStyle, s, KindColor, kindmsg)
         if conf.hasHint(hintSource) and info != unknownLineInfo:
-          conf.writeSurroundingSrc(info)
+          conf.writeSurroundingSrc(info, mColor)
```

Comment:
can you instead use `color.toMColor` and avoid having to duplicate color assignments above:
```nim
color = HintColor
mColor = mcHint
(etc)
```